_features/lin_advance.md

```diff
@@ -19,11 +19,43 @@ Since the root cause is pressure, `LIN_ADVANCE` de-couples extrusion from the ot
 ## Advantages
 
 - Better dimensional precision due to reduced bleeding edges.
-- Higher printing speeds are possible without any loss of print quality.
+- Higher printing speeds are possible without any loss of print quality - as long as your extruder can handle the needed speed changes.
 - Visible and tangible print quality is increased even at lower printing speeds.
 - No need for high acceleration and jerk values to get sharp edges.
 
 <br>
+
+## Special notes for v1.5
+
+### Changelog
+
+- K is now a meaningful value with the unit [mm of filament compression needed per 1mm/s extrusion speed] or [mm/mm/s].
+- Load inside stepper ISR reduced as no calculations are needed there any more. Instead, the extruder runs at a fixed speed offset during pressure adjustment. Therefore this version runs faster.
+- LIN_ADVANCE now respects hardware limitations set in Configuration.h, namely extruder jerk. If the pressure corrections would need faster adjusments than allowed by extruder jerk limit, the acceleration for this print move is limited to a value which allowes to use extruder jerk speed as the upper limit.
+- The pressure adjustment moves doesn't lead to a rattling extruder as it was in v1.0 as the extruder is now running at a smooth speed instead of jerking between multiples of extruder print speed.
+- This smooth extruder operation and respecting of jerk limits ensures no extruder steps are skipped.
+
+### New K value required
+
+As the unit of K has changed, you have to redo the K calibration procedure. See next chapter for details. While old v1 K values for PLA might be between 30-130, you can now expect K to be around 0.1-2.0.
+
+### LIN_ADVANCE can reduce your print acceleration
+
+In v1, if K was set to a high value which couldn't be handled by your printer , then the printer was losing steps and/or using all of it's processing power to execute extruder steps. In v1.5, this is handled much smarter. `LIN_ADVANCE` will now check if it can execute the advance steps as needed. If the needed extruder speed exceeds the extruder jerk limit, it will reduce the print acceleration for the line printed to a value which keeps the extruder speed within the limit.
```

Comment:
"handled by your printer , then the" --> "handled by your printer, then the" (removed the space before the comma)